src/phoenix/server/api/types/Span.py

```diff
@@ -291,6 +293,42 @@ async def contained_in_dataset(self, info: Info[Context, None]) -> bool:
         examples = await info.context.data_loaders.span_dataset_examples.load(self.id_attr)
         return bool(examples)
 
+    @strawberry.field(description="Invocation parameters for the span")  # type: ignore
+    async def invocation_parameters(self, info: Info[Context, None]) -> list[InvocationParameter]:
+        from phoenix.server.api.helpers.playground_clients import OpenAIStreamingClient
+        from phoenix.server.api.helpers.playground_registry import PlaygroundClientRegistry
+
+        db_span = self.db_span
+        attributes = db_span.attributes
+        llm_provider: GenerativeProviderKey = (
+            get_attribute_value(attributes, SpanAttributes.LLM_PROVIDER)
+            or GenerativeProviderKey.OPENAI
+        )
+        llm_model = get_attribute_value(attributes, SpanAttributes.LLM_MODEL_NAME)
+        invocation_parameters = get_attribute_value(
+            attributes, SpanAttributes.LLM_INVOCATION_PARAMETERS
+        )
+        if invocation_parameters is None:
+            return []
+        invocation_parameters = json.loads(invocation_parameters)
+        # find the client class for the provider, if there is no client class or provider,
+        # return openai as default
+        reg = PlaygroundClientRegistry()
```

Comment:
I think we can import the instantiated `PLAYGROUND_REGISTRY` singleton directly here rather than reinstantiating.

Reply:
Ah yeah, that's likely. I did this as a debugging step.

For what it's work, since PlaygroundClientRegistry is a singleton this is _technically_ fine if a little less performant.

Will try switching back anyway

Reply:
Done